operator/cmd/mesh/upgrade.go

```diff
@@ -192,9 +192,9 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l *Logger) (err error) {
 	if ysf, err = yamlFromSetFlags(currentSets, args.force, l); err != nil {
 		return err
 	}
-	currentProfileIOPSYaml, _, err := GenerateConfig(nil, ysf, args.force, nil, l)
+	currentProfileIOPSYaml, _, err := GenerateConfig(nil, ysf, true, nil, l)
```

Comment:
won't the warning still show up? And doesn't force do more than just remove the version check?

Reply:
The warning code is at: operator/cmd/mesh/profile-common.go: rewriteURLToLocalInstallPath(). The version check inside is used by all GenerateConfig() callers, and I think it is useful because it can prevent incompatible remote profile CR is used, e.g., using 1.4 built-in profile in 1.5 installation.

However, in the current master we support 1.5->1.6, i also makes sense that we remove the version check inside  rewriteURLToLocalInstallPath().